tail.go

```diff
@@ -45,8 +57,22 @@ func (s *Snitch) sendTail(aud *protos.Audience, pipelineID string, originalData
 			OriginalData:  originalData,
 			NewData:       postPipelineData,
 		}
-		tail.Ch <- tr
+
+		tail.ShipResponse(tr)
+	}
+}
+
+func (t *Tail) ShipResponse(tr *protos.TailResponse) {
+	// If we're sending too fast, drop the message
+	if time.Since(t.lastMsg).Milliseconds() < MinTailResponseIntervalMS {
+		// TODO: we should notify the snitch server that we're dropping messages somehow
+		// TODO: but this needs to be done in a way that only sends once in a while
+		t.log.Debugf("Dropping tail response for %s, too fast", tr.PipelineId)
```

Comment:
Let's do this as a `Warnf` for now.

We can also expose a new metric for "num dropped tail messages". We can display that in the UI somewhere?

Reply:
Good idea, done!